File: TwoSumII-InputArrayIsSorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] val = new int[2];
        int left = 0;
        int right = numbers.length-1;
        while(numbers[left] + numbers[right] != target){
            if(numbers[left] + numbers[right] < target){
                left++;
            }
            else{
                right--;
            }
        }
        val[0] = left+1;
        val[1] = right+1;
        return val;
    }
}

File: TwoSumII-InputArrayIsSorted.py
class Solution(object):
    def twoSum(self, num, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        left = 0
        right = len(num)-1
        while (num[left] + num[right] != target):
            if num[left] + num[right] < target:
                left += 1
            else:
                right -= 1
        return [left+1, right+1]
        